include/zephyr/logging/log_ctrl.h

```diff
@@ -95,6 +95,11 @@ __syscall void log_panic(void);
  */
 __syscall bool log_process(void);
 
+ /**
+  * @brief Process all pending log messages
+  */
+void log_flush(void);
```

Comment:
It should be possible to define `log_flush` as a syscall, to allow userspace to use it as well, not necessarily with LLEXT :slightly_smiling_face: